arm9/modules/18/asm/module_18.s

```diff
@@ -42930,7 +42930,7 @@ _0224E5CE:
 	ldr r0, _0224E6A0 ; =MOD18_0224E3F0
```

Comment:
what is this? this doesnt look like it uses normal calling conventions, what with the immediate r7 usage.

Reply:
Misdump, most likely.